fix(az): add fr-par-3 (#84)
scaleway/docs-content: webhosting/cpanel/reference-content.mdx

title: Reference Content
description: This page provides useful links and information for the management and configuration of your webhosting package, powered by cPanel 
dates:
  validation: 2021-09-08
  posted: 2021-09-12
---

<Message type="note">

> This documentation is directed toward all users who dispose of a webhosting package powered by cPanel on [Scaleway Dedibox](https://www.scaleway.com/en/dedibox/cloud-hosting/).

Reviewer: ```suggestion
This documentation is for users who have a webhosting package powered by cPanel on [Scaleway Dedibox](https://www.scaleway.com/en/dedibox/cloud-hosting/).
```
